Resources/Locale/en-US/_CD/admin/admin.ftl

```diff
@@ -2,3 +2,10 @@ admin-player-actions-window-cd-record-purge = Modify Character Records
 cd-actions-admin-modify-records = Modify Character Records
 cd-actions-admin-modify-reset = Reset Records
 cd-actions-admin-modify-del-entry = Delete Entry
+
+cd-eventpreferencepanel-title = Event Preference View
+cd-eventpreferencepanel-player = Player: {$player}
+cd-eventpreferencepanel-character = Character: {$characterName}
+
+cmd-eventpreferences-help = Usage: eventpreferences <name>
+cmd-eventpreferences-desc = Displays a player's current character's event preferences (i.e. antags)
```

Comment:
nitpick: can you add a cd prefix to these locale keys? These are generic enough that I could see upstream adding them.

Reply:
before the cmd part? because i think how it's currently formatted is how it's being referred to so that it's brought up in the console; if you mean more like cmd-cdeventpreferences-<stuff> then yeah can do

Reply:
Oh, right. I am moronic.